sdk/tests/LedgerIdTest.cc

```diff
@@ -128,15 +128,15 @@ TEST_F(LedgerIdTest, FromString)
 }
 
 //-----
-TEST_F(LedgerIdTest, FromBadStrings)
-{
-  // Given / When / Then
-  EXPECT_THROW(const LedgerId ledgerId = LedgerId::fromString("mainet"), std::invalid_argument);
-  EXPECT_THROW(const LedgerId ledgerId = LedgerId::fromString("this is a bad string"), std::invalid_argument);
-  EXPECT_THROW(const LedgerId ledgerId = LedgerId::fromString("1234567890abcdefg"), std::invalid_argument);
-  EXPECT_THROW(const LedgerId ledgerId = LedgerId::fromString("TESTNET"), std::invalid_argument);
-  EXPECT_THROW(const LedgerId ledgerId = LedgerId::fromString("Previewnet"), std::invalid_argument);
-}
+// TEST_F(LedgerIdTest, FromBadStrings)
+// {
+//   // Given / When / Then
+//   EXPECT_THROW(const LedgerId ledgerId = LedgerId::fromString("mainet"), std::invalid_argument);
+//   EXPECT_THROW(const LedgerId ledgerId = LedgerId::fromString("this is a bad string"), std::invalid_argument);
+//   EXPECT_THROW(const LedgerId ledgerId = LedgerId::fromString("1234567890abcdefg"), std::invalid_argument);
+//   EXPECT_THROW(const LedgerId ledgerId = LedgerId::fromString("TESTNET"), std::invalid_argument);
+//   EXPECT_THROW(const LedgerId ledgerId = LedgerId::fromString("Previewnet"), std::invalid_argument);
+// }
```

Comment:
Same with this?

Reply:
This test failed with the following error:

> Start 219: LedgerIdTest.FromBadStrings
219/361 Test #219: LedgerIdTest.FromBadStrings ..........................................................................................................................................................................................................................................................................................................................................................Subprocess aborted***Exception:   0.01 sec
Running main() from ./googletest/src/gtest_main.cc
Note: Google Test filter = LedgerIdTest.FromBadStrings
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from LedgerIdTest
[ RUN      ] LedgerIdTest.FromBadStrings
[       OK ] LedgerIdTest.FromBadStrings (0 ms)
[----------] 1 test from LedgerIdTest (0 ms total)
[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (0 ms total)
[  PASSED  ] 1 test.
[ FATAL ] ./googletest/include/gtest/internal/gtest-port.h:1774:: pthread_key_delete(key_)failed with error 22

You can see the failed PR check over [here](https://github.com/hashgraph/hedera-sdk-cpp/actions/runs/4575797534/jobs/8079092766).

My suggestion is the same like above - to leave this test commented and to create a separated issue for fixing it.